docs/advanced-run-method.md

```diff
@@ -0,0 +1,51 @@
+# Avanced run method options
```

Comment:
Is there another place the `cmd` and `nofail`, and `follow` arguments could go? This docs sections feels to me like it's mixing information of varying detail, utility, and knowledge back assumption. Specifically, the details here about `cmd`, `nofail`, and `follow` seem--to me, anyway--of fairly broad use and lacking requirement for any detailed control flow knowledge. The section about `shell` is denser, though, and with the inference capabilities from @michalstolarczyk  and @nsheff seems like it's less likely to need to be controlled directly (is the inference behavior still the default?)

Reply:
I don't understand what you're suggesting to do here. I was trying to just document in detail all the run options in one place... some are more common and simple than others, though. would you split them?

Reply:
Yeah sorry it was sort of rambly. It sounds like you safely navigated it and arrived at the meaning, though--I was saying it may be ideal to split them if possible, but not at all critical. I could just see people bogging down in the subprocess stuff and skipping then abandoning the reading before reading the other sections

Reply:
it's kind of mostly a reference section anyway so that's probably fine.

Reply:
I've done my best to correct this. I agree with you. I simplified the subprocess stuff, and moved it to the end.